File: Java/Node/src/Node/Biz/Interfaces/Task/IProcess.java
package Node.Biz.Interfaces.Task;

import java.rmi.RemoteException;

import Node.Biz.Custom.ProcParam;
/**
 * <p>This class create Submit Task Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */

public interface IProcess {
	  /**
	   * Execute
	   * @param parameters Input parameters
	   * @param param PostParam object
	   * @throws
	   * @return Status
	   */
  public String Execute(String[] parameters, ProcParam param);
}

File: EN Node for .NET environment/Node.Core/Biz/Interfaces/Task/IProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;

namespace Node.Core.Biz.Interfaces.Task
{
    /// <summary>
    /// The Interface for Task Process
    /// </summary>
    public interface IProcess
    {
        /// <summary>
        /// Process of Task.
        /// </summary>
        /// <param name="inputParameters">Task related parameters</param>
        /// <param name="param">Operation related Parameters</param>
        void Execute(string[] inputParameters, ProcParam param);
    }
}
